src/controllers/expenses.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  const expenses = expensesService.getAll();
+
+  res.send(expenses);
+};
+
+const getById = (req, res) => {
+  const { expensesId } = req.params;
+
+  const foundExpense = expensesService.getById(expensesId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const add = (req, res) => {
+  const { userId } = req.body;
+
+  const user = usersService.getById(userId);
+
+  if (!user) {
+    res.sendStatus(400);
```

Comment:
in this case the required parameter was passed, but the entity was not found. You should rather use 404 code here